src/iiiftilesource.js

```diff
@@ -370,7 +370,7 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
             if ( isv1 || levelWidth !== this.width ) {
```

Comment:
It's interesting how there does seem to be some logic for version 1, but not for 2 vs 2.1. Not sure if there is any way we could distinguish between `/2/context.json` and `/2.1/content.json`?

Reply:
As discussed offline, I don't see any differentiating values in the info.json to tell OSD the server does not support 2.1. 

Here is our info.json that eventually hits the 2.0 protocol
https://stor.stage.artstor.org/iiif/2019/09/20/16/8989d984-7e13-4002-8594-e786e929cfba.jp2/info.json

And the info.json for a image served by a 2.1 complain server seen by the `max` in the image request 
https://stacks.stanford.edu/image/iiif/hg676jb4964%2F0380_796-44/info.json
https://stacks.stanford.edu/image/iiif/hg676jb4964%2F0380_796-44/full/max/0/default.jpg

Reply:
good to at least know though. I think we got a good fix for the meantime though